src/status_im/contexts/settings/wallet/keypairs_and_accounts/import_seed_phrase/view.cljs

```diff
@@ -0,0 +1,71 @@
+(ns status-im.contexts.settings.wallet.keypairs-and-accounts.import-seed-phrase.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [status-im.common.enter-seed-phrase.view :as enter-seed-phrase]
+    [status-im.common.standard-authentication.core :as standard-auth]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn import-seed-phrase-controls
+  [{:keys [submit-disabled?
+           container-style
+           prepare-seed-phrase
+           seed-phrase
+           set-invalid-seed-phrase
+           focus-input]}]
+  (let [keypair             (rf/sub [:get-screen-params])
+        customization-color (rf/sub [:profile/customization-color])
+        show-errors         (rn/use-callback
+                             #(js/setTimeout
+                               (fn []
+                                 (focus-input)
+                                 (reagent/next-tick set-invalid-seed-phrase))
+                               600))
+        on-import-error     (rn/use-callback
+                             (fn [_error]
+                               (rf/dispatch [:hide-bottom-sheet])
+                               (show-errors)))
+        on-import-success   (rn/use-callback
+                             (fn []
+                               (rf/dispatch [:wallet/get-accounts])
+                               (rf/dispatch [:wallet/get-keypairs])
+                               (rf/dispatch [:hide-bottom-sheet])
+                               (rf/dispatch [:navigate-back])
+                               (rf/dispatch [:toasts/upsert
+                                             {:type  :positive
+                                              :theme :dark
+                                              :text  (i18n/label :t/key-pair-imported-successfully
+                                                                 {:name (:name keypair)})}]))
```

Comment:
So many dispatches here. Maybe we can dispatch a single event `:wallet/on-import-success` which would dispatch these events (also this logic not directly related to the view)

Reply:
Yeah this logic will be refactored before merging into a on-success event 👍 
I was waiting on another PR to merge so I could re-use some logic for this event.